apps/archive/commands.py

```diff
@@ -105,6 +106,10 @@ def _remove_expired_items(self, expiry_datetime):
                 logger.info('{} No items found to expire.'.format(self.log_msg))
                 return
 
+            if not touch(lock_name, expire=300):
```

Comment:
It means that for every iteration in the current loop we will reset expiry for the lock on 300 seconds, right?

Reply:
yep it would add 300 more every time, which might be actually a lot, will decrease it

Reply:
actually it doesn't add it, but rather sets expiry to now + 300, so it should be a bit more